src/ngSanitize/sanitize.js

```diff
@@ -158,7 +158,7 @@ function $SanitizeProvider() {
     return function(html) {
       var buf = [];
       htmlParser(html, htmlSanitizeWriter(buf, function(uri, isImage) {
```

Comment:
> Should that wait for a minor version bump?

Yes, as a breaking change this will be merged only into master (which will eventually become 1.7.0; probably some time around May).